### DISCUSSION TITLE
Welcome to drone-cloud-platform3.0 Discussions!

### DISCUSSION THREAD

**Discussion by waiting33118** [waiting33118/drone-cloud-platform3.0#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fitrifitriah** [comment]
hi im fitri fitriah, i want to visit the drone cloud platform website from the link you shared but when the signup doesn't work. i wonder why? thanks!
